mcts/mcts_policy.py

```diff
@@ -66,13 +67,16 @@ def update(self, rollout: TensorDictBase) -> None:
         target_value = self.value_estimator.value_estimate(value_estimator_input)
         target_value = target_value.squeeze(dim=0)
 
-        for idx in range(rollout.batch_size[0]):
-            state = rollout[idx, ...]
+        # usually time is along the last dimension (if envs are batched for instance)
+        steps = rollout.unbind(-1)
```

Comment:
Here I use -1 because I expect that we're unbinding along the time dimension (conventionally placed last in torchrl)